KMB/kmb_massload.py

```diff
@@ -169,34 +169,46 @@ def process_byline(entry):
         entry['byline'] = helpers.flip_name(entry['byline'])
 
 
-# @todo: update this per new copyright rules - T164568
 def process_license(entry):
     """
-    Identify the license and store back in entry.
+    Identify the license and store the back in entry.
 
     Don't include name/byline if unknown.
     """
+    entry['copyright'] = entry['copyright'].strip()
+    license_text = None
+
     if entry['license']:
         trim = 'http://kulturarvsdata.se/resurser/License#'
         entry['license'] = entry['license'].strip()[len(trim):]
-    # if not copyright = RAÄ and if license='' then, this is probably unfree
+
     if (entry['license'] == 'pdmark') or \
-            (entry['copyright'].strip() == 'Utgången upphovsrätt'):
-        entry['license'] = '{{PD-Sweden-photo}}'
-    elif (entry['license'] == 'by') or (entry['copyright'].strip() == 'RAÄ'):
-        # consider changing this to AND since there might be a cc-by image which isn't from RAA.
-        # Alternatively have another if inside which checks whethere copyright = RAA
-        param = '}}'
-        if (entry['byline'] == '{{unknown}}') or \
-                (entry['byline'] == '{{not provided}}'):
-            pass
-        else:
-            param = '|%s}}' % entry['byline']
-        entry['license'] = '{{CC-BY-RAÄ%s' % param
+            (entry['copyright'] == 'Utgången upphovsrätt'):
+        license_text = '{{PD-Sweden-photo}}'
```

Comment:
Do all the files marked as _pdmark_ or _Utgången upphovsrätt_ lack bylines? I'm looking at the documentation of this template, and it can apply to non-artistic photos taken before '69, but ironically it takes no parameters -- so if the photo belongs to this group but the author is known, you can't put it in there.

So we need to ensure that _pdmark_ or _Utgången upphovsrätt_ always means no byline, otherwise it will be lost.